src/PerfView/CommandLineArgs.cs

```diff
@@ -65,6 +65,7 @@ public static string GetHelpString(int maxLineWidth)
         // options common to multiple commands
         public string DataFile;             // This is the name of the ETL file (not the ZIP file)
         public string LogFile;
+        public string SymbolsAuth;          // Specifies authentication types for symbol servers (comma-separated)
```

Comment:
Rather than using a string and having to parse it, you can use the enum directly and even set the default value to be Interactive here.